.travis.yml

```diff
@@ -26,61 +26,43 @@
 
 ---
 language: python
+sudo: required
+dist: trusty
 
-matrix:
-    include:
-    - name: "1/4"
```

Comment:
Would it be OK seeing just `CHUNK=1/4` instead of just `1/4`? I added a global value in env. So not sure how to move it back to `matrix` with this new setting.

Reply:
This PR now uses `matrix` again, and has kept the `CHUNK` name. So resolving the discussion :D